### PR TITLE
Note that ISE theme is included in this extension

### LINKED ISSUE #1943: Consider moving the PowerShell ISE theme into a separate extension
* install this extension
* :bug: theme switches

This is really uncool - while I can understand your excitement about your theme you should understand that not everyone is equally excited about it. The extension already pops up a notification message and that would be a much more subtle place for "selling" your theme 

Screenshot 2019-05-03 at 12 39 41

### PR DESCRIPTION
**Pull request by andschwa**
Resolves #1943.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -3,13 +3,21 @@
...
+* [Part 1](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/06/debugging-powershell-script-in-visual-studio-code-part-1/)
+* [Part 2](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/13/debugging-powershell-script-in-visual-studio-code-part-2/)
+
+## Script analysis is reporting false errors
+
+Script analysis is provided by the [PSScriptAnalyzer] project on GitHub. If the warning
+message starts with `[PSScriptAnalyzer]` or if you are getting faulty script diagnostics
+(red and green squiggly lines under PowerShell in scripts) please [open an issue there].
+
+## Problems with syntax highlighting
```
Comment (rjmholt) [inline docs/troubleshooting.md:20]: Ah, this is no longer true

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -30,7 +38,19 @@ were, set:
 If you leave it enabled, you can customize the colors used for the various
 tokens. See [#3221] for more info and to leave suggestions.
 
-## VSCode is not working like the ISE
+## PSReadline features aren't working
+
+Integration of [PSReadline] into the PowerShell Integrated Console is under active
```
Comment (rjmholt) [inline docs/troubleshooting.md:43]: Also no longer true

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -45,7 +65,7 @@ VSCode itself and can't be changed by the extension.
 The VSCode maintainers are quite reasonable though,
 and you can ask for new features [in their repository](https://github.com/Microsoft/vscode).
 
-## Known Issues in the Extension
+## Known issues in the extension
```
Comment (rjmholt) [inline docs/troubleshooting.md:68]: We should review these

**Review by rjmholt — COMMENTED** [review]
The rest of this doc is still fairly out of date. We could change it in this PR or do it more incrementally, as you prefer.

File: docs/troubleshooting.md (hunk 1 above)
Comment (andschwa) [reply docs/troubleshooting.md:20]: PSSA or syntax highlighting?

File: docs/troubleshooting.md (hunk 3 above)
Comment (andschwa) [reply docs/troubleshooting.md:64]: Definitely 😄

File: docs/troubleshooting.md (hunk 1 above)
Comment (andschwa) [reply docs/troubleshooting.md:20]: Updated the latter with further clarification, since I'm quite sure script analysis is still PSSA's domain.

**Review by rjmholt — APPROVED** [review]

**Pull request merged**